meorg_client/tests/test_cli.py

```diff
@@ -1,10 +1,11 @@
+"""Test the CLI actions."""
```

Comment:
Updated all of the tests to make things work. Cleaned up docstrings, switched out for fixtures.